### DISCUSSION TITLE
W7DGP delivering a CubeSatSim seminar at SEAPAC Ham Convention in Seaside, OR June 3

### DISCUSSION THREAD

**Discussion by W7DGP** [alanbjohnston/CubeSatSim#269] [General]
My proposal for a CubeSat Sim seminar has been accepted for the SEAPAC convention and is scheduled for Saturday afternoon on June 3rd. I will be describing the project and my adventures in getting it going and all that I learned in the process. Of course, I plan to credit AMSAT and especially the support of Alan Johnston. I filled out the AMSAT logo permission form because it appears in my slides. My cubesat is still working perfectly after nearly one year in use. I hope to generate some interest in others taking on this project.  
I will be running the first school demo this fall at Seattle Academy, a private high school in Seattle. 
73, Larry W7DGP

**Comment by alanbjohnston** [comment]
Hi Larry, 

That’s great news! Thanks for sharing it with everyone!

Whenever I present on the CubeSatSim I always thank all the volunteer contributors to the project and mention all the open source software that is a key part of it. 

Best of luck with your presentation, and it would be great to hear back from you how it went.

73,
Alan 
KU2Y
